service/kubernetes/scripts/master.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/sh
 set -e
 
-kubeadm init --config /tmp/master-configuration.yml
+kubeadm init --config /tmp/master-configuration.yml \
+  --ignore-preflight-errors=Swap
```

Comment:
What is the actual reason kubernetes looks for swap is on in preflight-checks ?

Reply:
They say using swap is unsupported, but it has been working nicely in the past and I still believe we should enable it for the time being.